webapp/lib/view.dart

```diff
@@ -173,11 +173,41 @@ class BannerView {
   }
 }
 
+class ProjectSelectorView {
+  DivElement projectSelector;
+  SelectElement _projectOptions;
+
+  ProjectSelectorView() {
+    projectSelector = new DivElement()
+      ..id = 'project-selector';
+    _projectOptions = new SelectElement();
+    _projectOptions.onChange.listen((_) {
+      controller.command(controller.UIAction.needsReplyDataUpdated, null);
+    });
+    projectSelector.append(_projectOptions);
+  }
+
+  String get selectedProject => _projectOptions.value;
+
+  void populateProjects(Set<String> options) {
+    for(var option in options) {
+      if(_projectOptions.children.where((opt) => (opt as OptionElement).value == option).length > 0)
```

Comment:
nit: space needed between `for/if` and `(`

Reply:
Fixed that